docker/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: '3.7'
+
+networks:
+    emodb:
+        driver: bridge
+
+services:
+    zookeeper:
+        image: 'zookeeper:3.4.14'
+        networks:
+            - emodb
+        ports:
+            - '2181:2181'
+        volumes:
+            - 'zookeeper_data:/var/lib/zookeeeper/data'
+        environment:
+            - ALLOW_ANONYMOUS_LOGIN=yes
+    cassandra:
+        image: 'bazaarvoice/cassandra:2.2.4'
+        networks:
+            - emodb
+        build:
+            dockerfile: cassandra-Dockerfile
+            context: .
+        ports:
+            - '7000:7000'
+            - '7001:7001'
+            - '7199:7199'
+            - '9042:9042'
+            - '9160:9160'
+        volumes:
+            - 'cassandra_data:/var/lib/cassandra/data'
+        depends_on:
+            - zookeeper
+    kafka:
+        image: 'confluentinc/cp-kafka:5.3.1'
+        networks:
+            - emodb
+        ports:
+            - '9092:9092'
+        volumes:
+            - 'kafka_data:/var/lib/kafka/data'
+        environment:
+            - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
+            - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+            - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+            - ALLOW_PLAINTEXT_LISTENER=yes
+            - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+        depends_on:
+            - zookeeper
+    emodb-web:
+        image: 'bazaarvoice/emodb-web:latest'
```

Comment:
consider to build image on docker-compose up

Reply:
I kind of want to do the build as part of maven, but maybe in a separate PR.

Reply:
👍